hack/install-requirements.sh

```diff
@@ -20,7 +20,7 @@ echo "> Installing requirements"
 
 export GO111MODULE=on
 curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.27.0
-curl -s "https://raw.githubusercontent.com/helm/helm/v2.13.1/scripts/get" | bash -s -- --version 'v2.13.1'
+curl -s "https://raw.githubusercontent.com/helm/helm/master/scripts/get" | bash -s -- --version 'v2.13.1'
```

Comment:
Wouldn't it be better to simply clone that script instead of relying what is currently in the master branch?

Reply:
Hmm, I have seen several comments that recommend the usage of the install script from the master - ref https://github.com/helm/helm/issues/5486#issuecomment-474970334. I would rather rely it from the master instead of double maintenance on our side, or?

Reply:
I think the comment refers more to the fact that patches/fixes went obviously into master and not necessarily justified a new helm release (from their side).
Anyway, was just thinking out loud because of the usual security concerns with executing code from such "unstable" URLs (a one time "hack" of the helm repo and a one time sneak in of malicious code is enough to be run inside our CI/CD, even if helm maintainers would notice and take down a few hours later ...).
If there's at least a `helm/v3.x.y` version of the script fixing that issue, I'd prefer it, even more I'd prefer a copy of that script.
But we can defer this discussion as well because your current PR fixes the current CI/CD issues.

Reply:
```
curl -s "https://raw.githubusercontent.com/helm/helm/v3.2.4/scripts/get" | bash -s -- --version 'v2.13.1'
```
should also work

Reply:
I updated to `v2.16.9`.

Reply:
Thank you!